### DISCUSSION TITLE
Has 'children' with unique role assignments

### DISCUSSION THREAD

**Discussion by bajce** [pnp/pnpcore#634] [Q&A] [answered]
Is there any property which tells us weather on some of levels below the `List` or `ListItem` itself are objects with unique role assignments without loading all the objects from levels below? 
Similar like "Some content on this site has different permissions from what you see here." info on advance permission view in SharePoint Admin Center. 

P.S: Im using the `LoadItemsByCamlQuery` approach to load `ListItems` with `FolderServerRelativeUrl` set.

**Comment by jansenbe** [comment, marked as answer]
@bajce : unfortunately there's no API for that, enumerating what you're currently doing is the only approach.

**Reply by bajce** [reply]
@jansenbe: It looks like PnP-Site-Core had function `GetAllUniqueRoleAssignments` which did exactly what I need. It would be great if this function can be implemented in PnP Core SDK to provide more convenience and flexibility for developers.
